src/promptflow/promptflow/_sdk/_service/apis/collector.py

```diff
@@ -18,8 +19,30 @@
 from promptflow._sdk.entities._trace import Span
 
 
+# Check if the AML workspace is configured
+# If not, don't import any AML related modules
+def _is_workspace_configured():
+    subscription_id, resource_group_name, workspace_name = _get_workspace_info()
+    return subscription_id is not None and resource_group_name is not None and workspace_name is not None
+
+
+def _get_workspace_info():
+    # TODO: Change to workspace triad
+    return (
+        os.environ.get("pf_test_subscription_id"),
+        os.environ.get("pf_test_resource_group_name"),
+        os.environ.get("pf_test_workspace_name"),
+    )
+
+
 def trace_collector():
     content_type = request.headers.get("Content-Type")
+    if _is_workspace_configured():
+        from promptflow.azure._storage.cosmosdb.client import get_client
+
+        subscription_id, resource_group_name, workspace_name = _get_workspace_info()
+        span_client = get_client("Span", subscription_id, resource_group_name, workspace_name)
```

Comment:
try to make these constants defined in common constant file?

Reply:
Fixed~